src/gl_context.rs

```diff
@@ -65,16 +74,20 @@ impl<Native> GLContext<Native>
                attributes: GLContextAttributes,
                color_attachment_type: ColorAttachmentType,
                shared_with: Option<&Native::Handle>)
-        -> Result<GLContext<Native>, &'static str> {
-        Self::new_shared_with_dispatcher(size, attributes, color_attachment_type, shared_with, None)
+        -> Result<Self, &'static str> {
+        Self::new_shared_with_dispatcher(size,
```

Comment:
It... doesn't? `create_shared_with_dispatcher` calls the relevant method on the underlying platform-dependent context.